xcode/Subconscious/Shared/Components/Common/EntryList/EntryListView.swift

```diff
@@ -40,9 +40,7 @@ struct EntryListView: View {
                     // invisible marker to scroll back to
                     EmptyView().id(Self.resetScrollTargetId)
                     
-                    ForEach(entries.indices, id: \.self) { idx in
-                        let entry = entries[idx]
-                        
+                    ForEach(entries, id: \.self) { entry in
```

Comment:
We didn't need the index anymore, apparently

Reply:
Entries should be Identifiable, so we shouldn't need the second argument at all. That would be preferable, since `\.self` will consider value equality, and we would prefer to compare ID equality.